sentry_sdk/api.py

```diff
@@ -334,22 +337,33 @@ def flush(
     return get_client().flush(timeout=timeout, callback=callback)
 
 
-@scopemethod
 def start_span(
     **kwargs,  # type: Any
 ):
-    # type: (...) -> Span
-    return get_current_scope().start_span(**kwargs)
+    # type: (...) -> POTelSpan
+    global _potel_span_init_kwargs
+
+    if _potel_span_init_kwargs is None:
+        _potel_span_init_kwargs = inspect.signature(tracing.POTelSpan).parameters.keys()
```

Comment:
heavily against this kind of code, sorry
please write stupid code and list all the args, no introspection is necessary

Reply:
This kind of obfuscates what's going on -- I'd also prefer us to simply be stupidly explicit here since it makes it clearer what is happening.

Reply:
Fair enough, my reasoning was to avoid needing to update the arguments list in two places if it ever changes. But this way, it is perhaps less clear what is going on, so I will make this more explicit, as suggested.